packages/theming/atlas/src/themesource/atlas_core/native/core/helpers/maps.ts

```diff
@@ -38,6 +38,7 @@ export const mapsSquare = {
 export const mapsMaxSpace = {
     container: {
         flex: 1,
+        minHeight: 50,
```

Comment:
Is this for "When map size set to max space, it is important to set height of the surrounding container. Otherwise it wont be shown. Document this."?

Reply:
Yes! By using minHeight the user will always see (part of) the maps. So they know the widget is working correctly, it's just the styling that has to be adjusted to their needs